src/expo-plugin/withDangerouslyHandleAndroidSharedLibrary.ts

```diff
@@ -0,0 +1,58 @@
+import { ConfigPlugin, withAppBuildGradle } from '@expo/config-plugins';
+
+/**
+ * Add pickFirst libraries into app/build.gradle to fix build errors.
+ * This is a dangerous operation. It will override packagingOptions {} or android { packagingOptions {} } - use with caution as it might break your app
+ * [Learn more](https://github.com/mrousavy/react-native-vision-camera/pull/545)
+ * Credits: @mercyaj [Credits](https://github.com/mrousavy/react-native-vision-camera/pull/545#issuecomment-949477264)
+ */
+
+export const withDangerouslyHandleAndroidSharedLibrary: ConfigPlugin = (c) => {
+  return withAppBuildGradle(c, (config) => {
+    if (config.modResults.language === 'groovy') {
+      const body = `
+        pickFirst 'lib/**/lib*.so'
+      `;
+      const regexpPackagingOptions = /\bpackagingOptions\s*{[^}]*}/;
+      const packagingOptionsMatch = config.modResults.contents.match(regexpPackagingOptions);
```

Comment:
Probably would be worth while adding some tests for these cases since they're so dangerous, here is an example https://github.com/react-native-mapbox-gl/maps/blob/master/plugin/src/__tests__/withMapbox-test.ts

Reply:
We're going to remove this from the package and keep a link to a gist as reference